takoyaki/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -34,7 +37,25 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
-  private void configureButtonBindings() {}
+  private void configureButtonBindings() {
+    // Extend Intake
+    new JoystickButton(driveController, Constants.kIntake.ACUATE_INTAK)
+      .whenPressed(new InstantCommand(intake::actuateIntake, intake));
+
+    // Revert Intake
+    new JoystickButton(driveController, Constants.kIntake.RETRACT_INTAKE)
+      .whenPressed(new InstantCommand(intake::retractIntake, intake));
+      
+    // Run Intake
+    new JoystickButton(driveController, Constants.kIntake.RUN_INTACT)
+      .whenPressed(new InstantCommand(intake::startIntake, intake))
+      .whenReleased(new InstantCommand(intake::stopIntake, intake));
+
+    // Reserve Intake
+    new JoystickButton(driveController, Constants.kIntake.REVERSE_INTACT)
+      .whenPressed(new InstantCommand(intake::startReverse, intake))
+      .whenReleased(new InstantCommand(intake::stopIntake, intake));
```

Comment:
nit: keep the comments consistent with the code, like "extend intake" should be "actuate intake"